packages/backend/server/src/fundamentals/error/def.ts

```diff
@@ -103,6 +105,7 @@ export class UserFriendlyError extends Error {
     if (this.type !== 'internal_server_error') {
       // always record auth related error
       const isAuthError =
+        !ExcludedEvents.has(this.name) &&
```

Comment:
the stack checking logic could slow down the whole server. we can provide a whitelist beside `internal_server_error` to indicate those errors should be logged as well